code/src/App.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
 import data from './data.json'
 
+import Header from './Components/Header'
+import AlbumList from './Components/AlbumList';
+
+
 console.log(data)
```

Comment:
remember to remove console.logs  from your code after you finish debugging your code.